### PR TITLE
Issue #  25258 : recalculate line item pricing

### PR DESCRIPTION
**Pull request by discountscott**
Disconnect _shipto signal while populating... Then reconnect.

### CODE REVIEW

File: guiclient/salesOrder.cpp [hunk 1]
```diff
@@ -2623,6 +2623,8 @@ void salesOrder::populate()
     qu.exec();
     if (qu.first())
     {
+	  //disconnect the shipTo signal while we populate... Fixes recalculate pricing popping up
+	  disconnect(_shipTo, SIGNAL(newId(int)), this, SLOT(sParseShipToNumber()));
```
Comment (garyhgohoos) [inline guiclient/salesOrder.cpp]: i don't see how this change fixes the problem.  this change manipulates signals when populating a quote (not a sales order).  the probable cause of the problem is in the xwd salesOrder script.  this PR should be deleted

Comment (discountscott) [inline guiclient/salesOrder.cpp]: We were experiencing the same issue on quotes... This change stopped our quotes from generating the pop-up message when they shouldn't.

Comment (garyhgohoos) [inline guiclient/salesOrder.cpp]: the root cause of this problem is created by the xwd salesOrder script which is attempting to set the ship to based on the preferred selling site.  this change does not fix the root cause and does not fix the problem for Sales Orders and should be deleted.

**Comment by gpazo** [comment]
@garyhgohoos I agree, can you look at the real fix and link it to this pull request so @discountscott can see the solution?

File: guiclient/salesOrder.cpp (hunk 1 above)
Comment (garyhgohoos) [inline guiclient/salesOrder.cpp]: correct fix is private-extensions PR 561

**Comment by gilmoskowitz** [comment]
https://github.com/xtuple/private-extensions/pull/561

**Pull request closed without merging**